core/src/main/java/org/elasticsearch/plugins/InstallPluginCommand.java

```diff
@@ -193,17 +193,17 @@ protected void execute(Terminal terminal, OptionSet options, Environment env) th
     }
 
     // pkg private for testing
-    void execute(Terminal terminal, String pluginId, boolean isBatch, Environment env) throws Exception {
-        if (pluginId == null) {
-            throw new UserException(ExitCodes.USAGE, "plugin id is required");
+    void execute(Terminal terminal, String pluginName, boolean isBatch, Environment env) throws Exception {
```

Comment:
Ok my mistake, originally saw https://www.elastic.co/guide/en/elasticsearch/plugins/current/installation.html for install and thought 'name' was clearer for the output message.

Should these commits be squashed?